images/origin/scripts/recycler.sh

```diff
@@ -22,8 +21,20 @@ if [[ ! -d ${dir} ]]; then
     exit 1
 fi
 
+# remount to nfsv3 so not restricted by nfs v4 acls, see nfs4_acl(5)
```

Comment:
I'm going to edit the docs to say this is currently a requirement to use the recycler.

The next solution is to allow many kinds of recyclers for many kinds of volumes. Users should be able to configure them as they will, using specific service accounts, and we divest ourselves of the responsibility of trying to figure this out for all use cases.